plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -86,10 +97,15 @@ public CheckpointWriter(TypeManager typeManager, CheckpointSchemaManager checkpo
 
     public void write(ConnectorSession session, CheckpointEntries entries, Path targetPath)
     {
+        Map<String, String> configuration = entries.getMetadataEntry().getConfiguration();
+        boolean writeStatsAsJson = Boolean.parseBoolean(configuration.getOrDefault(DELTA_CHECKPOINT_WRITE_STATS_AS_JSON_PROPERTY, "true"));
+        // The default value is false in https://github.com/delta-io/delta/blob/master/PROTOCOL.md#checkpoint-format, but Databricks defaults to true
```

Comment:
here is some context information i stumbled upon:

https://docs.databricks.com/delta/optimizations/file-mgmt.html#manage-column-level-statistics-in-checkpoints